icssc-projects/peterportal-client
# Connecting review components to backend

# Connecting review components to backend

## Description


## Screenshots
### Review Form
![Screen Shot 2021-02-26 at 17 47 28](https://user-images.githubusercontent.com/17202423/109371645-b7bdef80-785a-11eb-9bb7-05b7c13b8e42.png)
### Reviews
|Before|After|
|--|--|
|![Screen Shot 2021-02-26 at 17 48 13](https://user-images.githubusercontent.com/17202423/109371671-d15f3700-785a-11eb-9a50-8158eb8b9c07.png)|![Screen Shot 2021-02-26 at 17 47 19](https://user-images.githubusercontent.com/17202423/109371642-b5f42c00-785a-11eb-96c8-ba1a836d6e7d.png)|

## Steps to verify/test this change:
1. `git pull reviews-api`
2. Once PeterPortal opens up, go to the ICS32A course page reviews
3. Upvote/downvote any review and refresh the page to make sure the score has been updated
4. Add a review through the form and make sure the reviews page get updated to show it

## Todos:
- [ ] Better way to alert user that they're over the 500 character limit
